coordinator/src/main/scala/filodb.coordinator/QueryActor.scala

```diff
@@ -55,6 +57,14 @@ final class QueryActor(memStore: MemStore,
 
   val config = context.system.settings.config
 
+  var filodbSpreadMap = new mutable.HashMap[String, Int]()
+  config.getConfig("filodb.spread.override").entrySet().asScala.
+    foreach { x => filodbSpreadMap.put(x.getKey(), x.getValue().unwrapped().asInstanceOf[Int])}
+  val applicationShardKeyName = dataset.options.nonMetricShardColumns(0)
```

Comment:
We need to think about how to have multiple keys in here. This requirement will come soon and better to address right away. Lets talk in person.